README.md

```diff
@@ -1,34 +1,30 @@
+Browse all current and draft EIPs on [the official EIP site](http://eips.ethereum.org/).
 
-Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft EIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your EIP contains either your Github username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
+## Project Goal
 
-When you believe your EIP is mature and ready to progress past the draft phase, you should do one of two things:
+The Ethereum Improvement Proposals repository exists as a place to discuss concrete proposals with potential users of the proposal and the Ethereum community at large.
 
- - **For a Standards Track EIP of type Core**, ask to have your issue added to [the agenda of an upcoming All Core Devs meeting](https://github.com/ethereum/pm/issues), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the EIP editors will update the state of your EIP to 'Accepted'.
- - **For all other EIPs**, open a PR changing the state of your EIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the EIP - they may close the PR and request that you fix the issues in the draft before trying again.
+We promote high-quality, peer-reviewed proposals in two categories:
 
-# EIP Status Terms
-* **Draft** - an EIP that is undergoing rapid iteration and changes
-* **Last Call** - an EIP that is done with its initial iteration and ready for review by a wide audience
-* **Accepted** - a core EIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author
-* **Final (non-Core)** - an EIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
-* **Final (Core)** - an EIP that the Core Devs have decide to implement and release in a future hard fork or has already been released in a hard fork
-* **Deferred** - an EIP that is not being considered for immediate adoption. May be reconsidered in the future for a subsequent hard fork.
+- [Consensus Changes](EIPS/eip-1.md#eip-types) require people running an Ethereum client to update their software to implement the change. If enough people do not upgrade then an existing network will fork into two separate networks.
+- [Other Proposals](EIPS/eip-1.md#eip-types) do not require a change in client software and can be used by any interested party.
```

Comment:
Same comment as above, both these links go to the same place, despite hinting that they go to different places.  Consider moving the leak to the lead-in sentence:
> ... proposals in two [categories](EIPS/eip-1.md#eip-types):

Reply:
I have higher expectations for a future EIP-1. But unlike change the all.html, I will defer editing EIP-1 at this time. 

I request a slight exception to the UX rule in this case.